store/sqlstore/command_store.go

```diff
@@ -64,7 +66,14 @@ func (s SqlCommandStore) Save(command *model.Command) (*model.Command, *model.Ap
 func (s SqlCommandStore) Get(id string) (*model.Command, *model.AppError) {
 	var command model.Command
 
-	if err := s.GetReplica().SelectOne(&command, "SELECT * FROM Commands WHERE Id = :Id AND DeleteAt = 0", map[string]interface{}{"Id": id}); err != nil {
+	sql, args, err := s.getQueryBuilder().
+		Select("*").
+		From("Commands").
```

Comment:
Just a nonblocking suggestion, this bit could be abstracted on its own `commandsQuery`, same way as you did in https://github.com/mattermost/mattermost-server/pull/14549

Reply:
Done!